### PR TITLE
Issue 99: Informative messaging for priors

### LINKED ISSUE #99: Add informative messaging (for priors minimum)
> Perhaps I could add a message in epidist_priors saying "using default priors"?

Suggest "informative" messaging is its own issue. We use `cli` in `{epinowcast}` and I like it.

_Originally posted by @seabbs in https://github.com/epinowcast/epidist/issues/95#issuecomment-2160160720_

Also should think about other places in the package we would like more messaging. Could start looking through `epinowcast` and copy places messaging is provided there.

### PR DESCRIPTION
**Pull request by athowes**
## Description

This PR closes #99.

## Checklist

- [ ] My PR is based on a package issue and I have explicitly linked it.
- [ ] I have included the target issue or issues in the PR title in the for Issue(s) *issue-numbers*: PR title
- [ ] I have read the [contribution guidelines](https://github.com/epinowcast/.github/blob/main/CONTRIBUTING.md).
- [ ] I have tested my changes locally.
- [ ] I have added or updated unit tests where necessary.
- [ ] I have updated the documentation if required.
- [ ] My code follows the established coding standards.
- [ ] I have added a news item linked to this PR.
- [ ] I have reviewed CI checks for this PR and addressed them as far as I am able.

### CODE REVIEW

File: R/latent_individual.R [hunk 1]
```diff
@@ -205,12 +205,21 @@ epidist_family.epidist_latent_individual <- function(data, family = "lognormal",
...
+#' @importFrom cli cli_inform
 #' @export
 epidist_prior.epidist_latent_individual <- function(data, ...) {
   validate_latent_individual(data)
 
   prior1 <- brms::prior("normal(2, 0.5)", class = "Intercept")
   prior2 <- brms::prior("normal(0, 0.5)", class = "Intercept", dpar = "sigma")
+
+  cli::cli_inform(c(
+    "i" = "The following priors have been set:",
```
Comment (seabbs) [inline R/latent_individual.R:217]: I think you need to be what the link function is for each or this is confusing.

**Review by seabbs — COMMENTED** [review]
I'm not sure I would want this level of verbosity but I am happy to go with it if you think its needed. One small issue with needing to define what the link is in order for people to understand what they are looking at.

**Comment by athowes** [comment]
Yes I also think it's a bit verbose. What about an option `verbose = FALSE` and have it only show if set to `TRUE`?

**Comment by seabbs** [comment]
> Yes I also think it's a bit verbose

We should be able to use`cli` so it only shows once per session or once every 8 hours etc. Perhaps that is the way to go?

**Comment by athowes** [comment]
Give message about prior on the transformed more natural scale.

File: R/latent_individual.R (hunk 1 above)
Comment (athowes) [reply R/latent_individual.R:217]: I have written something corresponding more directly to what the `brms` prior is. I think that we still have some confusion about the priors and families which we will need to handle (e.g. #131, #171). So I'd be in favour of this PR just saying what there is at the moment, and it's something we can come back to once we sort out how we are handling families.

**Comment by athowes** [comment]
I've added an issue about the additional complexities here: https://github.com/epinowcast/epidist/issues/172

I've added the "display once every 8 hours". It's also possible to do "display on new session" instead if people have a preference.

**Review by seabbs — APPROVED** [review]

**Pull request merged**